### DISCUSSION TITLE
Home Assistant integration

### DISCUSSION THREAD

**Discussion by arboeh** [BlazejosP/huawei-sun2000-API-CLI#29] [Ideas]
As I understood this is the "new" connection method to connect to the SUN2000. Modbus TCP is deprecated?
So it would be great to see this project as Home Assistant integration.

**Comment by BlazejosP** [comment]
No this is a other method to connect to your Inverter. First with Modbus TCP is working correctly but need an hardware. Dedicated Wi-Fi Access point and then with use of software from other projects you can access to the data directly. Here in that project you connect to Huawei cloud account on their server in Holland and from there you can download your data. You don't need extra hardware but you are dependant on Huawei service and internet connection.

If you are interested in Home Assistant integration in not so long time I plan to add integration with MQTT so that my solve problem and will work without problem with any home automation software. If you will be interested to write some code in bash and add to repository that my speed up such integration?
